src/config/shared.js

```diff
@@ -32,3 +32,38 @@ export function extractBucketMetadata(transformedData) {
   etc.bucketSortOrder = sorter;
   return etc;
 }
+
+export const facets = {
+  activeBuckets: {
+    isMulti: true,
+    defaultValue: [],
+  },
+  aggregationLevel: {
+    defaultValue: 'build_id',
+  },
+  channel: {
+    defaultValue: 'nightly',
+  },
+  os: {
+    defaultValue: 'Windows',
+  },
+  process: {
+    defaultValue: 'content',
+  },
+  product: {
+    defaultValue: 'firefoxDesktop',
```

Comment:
Just a caveat... the Glean name for Firefox will be `firefox.desktop`, if I saw that correctly. The API currently expects just `firefox`. I'm happy to change it to something that is more clearly "telemetry firefox" vs Glean firefox.

Reply:
Would you mind opening a follow-up issue for this?